src/Form/__snapshots__/form.test.js.snap

```diff
@@ -267,7 +267,7 @@ exports[`Form component sanity matches themed snapshot 1`] = `
           width="1"
         />
         <svg
-          class="icon-valid c8"
+          class="c8 icon-valid"
```

Comment:
The other changes are switching the order of the class names. Not sure what caused this.